src/com/xilinx/rapidwright/interchange/PhysNetlistReader.java

```diff
@@ -521,6 +523,32 @@ protected void readRouting(StructList.Reader<PhysNet.Reader> nets, Consumer<Net>
                 net.addPIP(pip);
             }
             stubWires.clear();
+
+            // Nets with more than one routed source (e.g. A_O and AMUX) should have
+            // the first primary source PIP marked as a logical driver
+            SitePinInst altSource = net.getAlternateSource();
```

Comment:
Should we avoid running this code for the nets `GLOBAL_LOGIC[01]`?  At the least, it might help runtime.

Reply:
Good spot! f6b92f9 makes it only operate on wire nets (not either static, nor the used net) and assert that it is not a clock net either.